Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM osawagiboy/slimane-docker:latest
```

Comment:
Cold you change the repo to slimane/application something like that?
I wanna use that as official.